src/main/java/com/util/ServletUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.util;
+
+import com.entities.Organization;
+import com.google.gson.Gson;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public final class ServletUtil {
+
+    public static <T> T expect(HttpServletRequest request , Class<T> Class) throws IOException {
```

Comment:
We can narrow the Exception. (`JsonParseException` is a subclass of `IOException`) We should always throw the lowest level of exception.
```suggestion
    public static <T> T expect(HttpServletRequest request , Class<T> Class) throws JsonParseException {
```

Reply:
we can change the order of the parameter list as well. 
`(Class<T> Class, HttpServletRequest request )`

Sample usage: 
`ServletUtil.expect(Organization.class, request);`  this will be read as expect an organisation from the request.
Our code should always be readble. It's something similar to writing a story. ;)

Reply:
I tried using this JsonParseException but it got an error like this. Any idea how I can solve this?

![image](https://user-images.githubusercontent.com/77311602/135125656-6c4b3290-2ed1-475b-bfc4-d823031a009d.png)